creator-node/src/services/stateMachineManager/stateMachineUtils.js

```diff
@@ -267,56 +267,63 @@ const makeMetricToRecord = (
 
 const makeQueue = ({
   name,
+  processor,
+  logger,
   removeOnComplete,
   removeOnFail,
-  lockDuration,
-  prometheusRegistry = null,
-  limiter = null
+  prometheusRegistry,
+  concurrency = 1,
+  limiter = null,
+  onFailCallback = null
 }) => {
-  // Settings config from https://github.com/OptimalBits/bull/blob/develop/REFERENCE.md#advanced-settings
-  const queue = new BullQueue(name, {
-    redis: {
-      host: config.get('redisHost'),
-      port: config.get('redisPort')
-    },
+  const connection = {
+    host: config.get('redisHost'),
+    port: config.get('redisPort')
+  }
+  const queue = new Queue(name, {
+    connection,
     defaultJobOptions: {
       removeOnComplete,
       removeOnFail
-    },
-    settings: {
-      // Should be sufficiently larger than expected job runtime
-      lockDuration,
-      // We never want to re-process stalled jobs
-      maxStalledCount: 0
```

Comment:
you removed the `settings` param entirely?

Reply:
retrying stalled jobs is a deviation from our previous logic so we should consider this a bit - can discuss